BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -267,3 +263,38 @@ fileprivate final class ScrollerWrapperView: UIView {
     }
 
 }
+
+private extension ScrollerWrapperView {
+
+    final class CustomScrollView: UIScrollView {
+
+        private var _refreshControl: UIRefreshControl? = nil
+
+        override var refreshControl: UIRefreshControl? {
+            get {
+                if #available(iOS 10.0, *) {
```

Comment:
Thought: We've deprecated iOS 9 in SPOS, so we can support iOS 10+ here. Shall we bump the min deployment target and remove this special casing?